libraries/SPIFFS/src/SPIFFS.cpp

```diff
@@ -1,32 +1,26 @@
-// Copyright 2015-2016 Espressif Systems (Shanghai) PTE LTD
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-#include "vfs_api.h"
-
 extern "C" {
 #include <sys/unistd.h>
 #include <sys/stat.h>
 #include <dirent.h>
 #include "esp_spiffs.h"
 }
+
 #include "SPIFFS.h"
 
 using namespace fs;
 
-SPIFFSFS::SPIFFSFS(FSImplPtr impl)
-    : FS(impl)
-{}
+SPIFFSImpl::SPIFFSImpl()
+{
+}
+
+bool SPIFFSImpl::exists(const char* path) {
```

Comment:
please use a new line before the curly braces. I see this a few more times below :)